ftplugin/org.vim

```diff
@@ -115,6 +115,7 @@ for p in vim.eval("&runtimepath").split(','):
 from orgmode._vim import ORGMODE, insert_at_cursor, get_user_input, date_to_str
 ORGMODE.start()
 
+from Date import Date
```

Comment:
I made a mistake and removed it before, I haven't realised that it was used in calendar plugin... I somehow missed that one for a long time.